Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM google/cloud-sdk:170.0.1-slim
```

Comment:
Dockerfile was added because I prefer to have the environment that way, but feel free to drop this from this PR

Reply:
If we want docker I suggest we setup a dockerhub hook to auto build this image